packages/react-core/src/components/Dropdown/examples/Dropdown.md

```diff
@@ -20,81 +20,134 @@ import avatarImg from '../../Avatar/examples/avatarImg.svg';
 
 ### Basic
 
+Basic dropdowns present all selectable items in a menu upon clicking within the dropdown box. The `isOpen` property is used to flag a menu as opened. 
+
+All of the items in a dropdown list are created using the `DropdownItem` component. The following example shows a few different states of dropdown items. For each item, you may: 
+- Use the `tooltip` property to enable tooltip support. 
+- Use the `isDisabled` property to disable certain menu items.
+- Use the `isAriaDisabled` property to 
+
+This example also uses a `DropdownSeparator` to add a horizontal line to separate sections of the menu. 
+
 ```ts file='./DropdownBasic.tsx'
 ```
 
 ### With initial selection
 
+Use the `autoFocus` property to indicate a dropdown item that is pre-selected upon clicking the dropdown box. `autoFocus` cannot be applied to disabled dropdown items.
+
 ```ts file="./DropdownInitialSelection.tsx"
 ```
 
 ### With groups
 
+Use the `DropdownGroup` component to group sets of dropdown items together. This allows you to group related items together to help contextualize the options. Use the `isGrouped` property on the `Dropdown` component to indicate that the dropdown contains grouped items.
+
 ```ts file="./DropdownGroups.tsx"
 ```
 
 ### Disabled
 
+Use the `isDisabled` property within the `DropdownToggle` component to disable the expansion of the entire dropdown menu.
+
 ```ts file="./DropdownDisabled.tsx"
 ```
 
+### Dropdown with descriptions
+
+Assign a `description` property and respective value to each `DropdownItem` that you wish you offer more context to. This description will appear in the menu itself, under the dropdown item it corresponds to.
+
+```ts file="./DropdownDescriptions.tsx"
+```
+
+### Dropdown with image and text
+
+Use the `icon` property to display an image in the `DropdownToggle`. Icons can be used alongside text, as shown in the following example.
+
+```ts file="./DropdownImageAndText.tsx"
+```
+
 ### Primary toggle
 
+You can adjust the styling of the `DropdownToggle` as needed. The following example uses the `isPrimary` to apply primary button styling. This method is deprecated and we now suggest using `toggleVariant="primary"` to acheive the same results.
+
```

Comment:
I will open a follow up issue to use our recommended approach in the example. That way you can remove the reference to the `isPrimary` prop in this example.

Reply:
Little type here - acheive instead of achieve

Reply:
This ^ opened issue has been addressed, so now the example no longer uses the 'isPrimary' prop.

Reply:
Noted! Will remove the isPrimary reference in that case